README.md

```diff
@@ -1,4 +1,87 @@
-# Hello_Microverse-Pro
+# Hello-Microverse-Project
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+Hello-Microverse-Project
+
+> Describe your project in 1 or 2 sentences.
+Hello-Microverse-Project is a my first project after the enrollment with Microverse.This is the Project about basic setup of Repo in github with files such as README.md,.gitignore,linters.yml file and basic setup of HTML and CSS file.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+This Project is built with HTML and CSS.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Demo Project of Microverse]**
+- **[Getting familiar with Github,HTML,CSS]**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](http://127.0.0.1:5500/index.html)
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
```

Comment:
[OPTIONAL]
- [ ] Kindly delete any instructions since they were only for your guidance, leaving them makes your `README` looks unprofessional.